flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/batch/sql/TableScanTest.scala

```diff
@@ -34,4 +34,22 @@ class TableScanTest extends TableTestBase {
     util.verifyPlan("SELECT * FROM MyTable")
   }
 
+
+  @Test
+  def testDDLTableScan(): Unit = {
+    TestCollectionTableFactory.isStreaming = false
+    util.addTable(
+      """
+        |CREATE TABLE src (
+        |  ts TIMESTAMP(3),
+        |  a INT,
+        |  b DOUBLE,
+        |  WATERMARK FOR ts AS ts - INTERVAL '0.001' SECOND
+        |) WITH (
+        |  'connector' = 'COLLECTION'
+        |)
+      """.stripMargin)
+    util.verifyPlan("SELECT * FROM src WHERE a > 1")
+  }
```

Comment:
Should we put the watermark info in the digest ? because you push the information into that.

Reply:
For batch mode, we shouldn't put watermark info in the digest, because no watermark in batch mode. This is test is used to verify no watermark info in the digest.